ocdskingfisherprocess/transform/compile_releases.py

```diff
@@ -55,18 +56,112 @@ def get_ocids(self):
             for row in query:
                 ocids.append(row['ocid'])
 
+        # get ocids from records
+        with self.database.get_engine().begin() as engine:
+            query = engine.execute(
+                sa.text(
+                    " SELECT r.ocid FROM record AS r" +
+                    " LEFT JOIN compiled_release AS cr ON " +
+                    " cr.ocid = r.ocid and cr.collection_id = :destination_collection_id" +
+                    " WHERE r.collection_id = :collection_id and cr.ocid is NULL" +
+                    " GROUP BY r.ocid "
+                ),
+                collection_id=self.source_collection.database_id,
+                destination_collection_id=self.destination_collection.database_id
+            )
+
+            for row in query:
+                if row['ocid'] not in ocids:
+                    ocids.append(row['ocid'])
+
+        # done
         return ocids
 
-    def process_ocid(self, ocid):
+    def _process_ocid(self, ocid):
+
+        # Load Records
+        records = []
+        with self.database.get_engine().begin() as engine:
+            query = engine.execute(sa.text(
+                " SELECT record.* FROM record " +
+                " WHERE record.collection_id = :collection_id AND record.ocid = :ocid "
+            ), collection_id=self.source_collection.database_id, ocid=ocid)
+
+            for row in query:
+                records.append(self.database.get_data(row['data_id']))
+
+        # Decide what to do .....
+        if len(records) > 1:
+
+            # This counts as taking a random record, as we have not ordered the SQL query by anything.
+            # (In practice the way postgres works I think we will always get the first record by load order,
+            #     but with no ORDER BY clause that is not guaranteed)
+            warning = 'There were multiple records for this OCID! ' + \
+                    'We have picked one at random and passed it through this transform.'
```

Comment:
Change warning:

* were -> are
* We have picked one at random … -> The record to pass through was selected arbitrarily.

And remove the comment. Same change for similar warning below.

In general, for all messages:

* Use present tense.
* Use passive voice instead of "we".